diff --git a/node_modules/vitepress/dist/node/serve-61783397.js b/node_modules/vitepress/dist/node/serve-61783397.js
index 2f64241..2e4afb8 100644
--- a/node_modules/vitepress/dist/node/serve-61783397.js
+++ b/node_modules/vitepress/dist/node/serve-61783397.js
@@ -36615,7 +36615,7 @@ function createVitePressPlugin(root, siteConfig, ssr = false, pageToHashMap, cli
     },
     renderStart() {
       if (hasDeadLinks) {
-        throw new Error(`One or more pages contain dead links.`);
+        // throw new Error(`One or more pages contain dead links.`);
       }
     },
     configureServer(server) {
