diff --git a/node_modules/vitepress/dist/node/build/render.js b/node_modules/vitepress/dist/node/build/render.js
index e32673a..c5d28de 100644
--- a/node_modules/vitepress/dist/node/build/render.js
+++ b/node_modules/vitepress/dist/node/build/render.js
@@ -75,12 +75,14 @@ function resolvePageImports(config, page, result, indexChunk) {
     // they are start fetching as early as possible
     const srcPath = vite_1.normalizePath(fs_extra_1.default.realpathSync(path_1.default.resolve(config.srcDir, page)));
     const pageChunk = result.output.find((chunk) => chunk.type === 'chunk' && chunk.facadeModuleId === srcPath);
-    return Array.from(new Set([
+    return Array.from(new Set(
+      [
         ...indexChunk.imports,
         ...indexChunk.dynamicImports,
-        ...pageChunk.imports,
-        ...pageChunk.dynamicImports
-    ]));
+        ...(pageChunk?.imports || []),
+        ...(pageChunk?.dynamicImports || [])
+      ].filter(Boolean)
+    ));
 }
 function renderHead(head) {
     return head
